redisdb/tests/test_e2e.py

```diff
@@ -110,13 +85,6 @@ def test_e2e_v_latest(dd_agent_check, master_instance):
     assert_common_metrics(aggregator)
 
     tags = ['redis_host:{}'.format(common.HOST), 'redis_port:6382', 'redis_role:master']
-    aggregator.assert_metric('redis.mem.overhead', count=2, tags=tags)
-    aggregator.assert_metric('redis.mem.startup', count=2, tags=tags)
-    aggregator.assert_metric('redis.active_defrag.running', count=2, tags=tags)
-    aggregator.assert_metric('redis.active_defrag.hits', count=2, tags=tags)
-    aggregator.assert_metric('redis.active_defrag.misses', count=2, tags=tags)
-    aggregator.assert_metric('redis.active_defrag.key_hits', count=2, tags=tags)
-    aggregator.assert_metric('redis.active_defrag.key_misses', count=2, tags=tags)
```

Comment:
Why are we dropping this bunch of metrics?

Reply:
We are not, there were multiple tests here and now there is only one. So all metrics are now defined on a single place:
* redis.mem.overhead: line 62
* redis.mem.startup: 54
* redis.active_defrag.running: 55
* redis.active_defrag.hits: 56
* redis.active_defrag.misses: 57
* redis.active_defrag.key_hits: 58
* redis.active_defrag.key_misses: 59
* redis.server.io_threads_active: 89
* redis.stats.io_threaded_reads_processed:90
* redis.stats.io_threaded_writes_processed: 91
* redis.cpu.sys_main_thread: 93
* redis.cpu.user_main_thread: 94

For this PR it might be worth setting a side-by-side diff on github